src/backend/renderer/multigpu/mod.rs

```diff
@@ -723,16 +723,16 @@ impl<'render, 'target, R: GraphicsApi, T: GraphicsApi> fmt::Debug for MultiRende
     }
 }
 
-impl<'render, 'target, R: GraphicsApi, T: GraphicsApi> AsRef<<R::Device as ApiDevice>::Renderer>
-    for MultiRenderer<'render, 'target, R, T>
+impl<R: GraphicsApi, T: GraphicsApi> AsRef<<R::Device as ApiDevice>::Renderer>
+    for MultiRenderer<'_, '_, R, T>
 {
     fn as_ref(&self) -> &<R::Device as ApiDevice>::Renderer {
         self.render.renderer()
     }
 }
 
-impl<'render, 'target, R: GraphicsApi, T: GraphicsApi> AsMut<<R::Device as ApiDevice>::Renderer>
-    for MultiRenderer<'render, 'target, R, T>
```

Comment:
I am not sure I like this clippy lint telling us to remove explicitly named lifetimes. The names are descriptive.. I guess it is fine, as long as it is only doing that for `impl`s though?

Reply:
Good point. But yeah, I'm not sure the named lifetimes add anything in things like `AsRef`/`AsMut` impls, where the lifetime doesn't appear anywhere in the body. So it's probably good here.